src/storage/StorageFlags.cpp

```diff
@@ -26,6 +26,9 @@ DEFINE_int32(rebuild_index_batch_num, 256,
 DEFINE_int32(rebuild_index_locked_threshold, 256,
              "The locked threshold will refuse writing.");
 
+DEFINE_int32(rebuild_index_process_interval, 30000,
```

Comment:
3ms is enough I think? 30ms means there are 33 request at most for one second.

Reply:
> 3ms is enough I think? 30ms means there are 33 request at most for one second.

good catch. fixed.